vmdb/app/controllers/api_controller/accounts.rb

```diff
@@ -4,7 +4,7 @@ module Accounts
     # Accounts Subcollection Supporting Methods
     #
     def accounts_query_resource(object)
-      object.send("accounts")
+      object.accounts
```

Comment:
@chessbyte To piggy back on @Fryguy's comments, it's not clear if `object` here has a private method `accounts` which is why the send was here or the original code was doing a useless send. 

Are you saying you verified all of the non-`self.send` calls are all public?